src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  display: flex;
+  width: 200px;
+
+  &__tile {
+    display: flex;
+    flex-direction: column;
+    height: $card-height;
+    width: $card-width;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background-color: #fff;
+    padding: 0 $card-inner-indent;
+  }
+
+  &__image {
+    display: flex;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    flex-shrink: 0;
+
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    color: $Main-accent;
+    font-family: "Roboto", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    color: $Secondary-accent;
+    font-family: "Roboto", sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 17px;
+  }
+
+  &__stars {
+    width: 96px;
+    height: 16px;
+    flex-shrink: 0;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    flex-shrink: 0;
+
+    color: $Main-accent;
+    text-align: right;
+    font-family: "Roboto", sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+    line-height: 18px;
+
+    &-label {
+      color: $Secondary-accent;
+      font-family: "Roboto", sans-serif;
+      font-size: 12px;
+      font-weight: 400;
+    }
+
+    &-amount {
+      color: $Main-accent;
+      text-align: right;
+      font-family: "Roboto", sans-serif;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__buy {
+    height: 40px;
+    width: 100%;
+    background-color: $Blue-accent;
+    border-radius: 5px;
+    margin: $card-inner-indent 0;
+
+    color: white;
+    text-align: center;
+    text-decoration: none;
+    font-family: "Roboto", sans-serif;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 40px;
+    text-transform: uppercase;
+
+    &:hover {
+      box-sizing: border-box;
+      border-radius: 5px;
+      border: 1px solid $Blue-accent;
```

Comment:
Don't add a new border on hover. Add the transparent border for the default state of the element and just change its color on hover